src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -21,6 +21,8 @@
  * @author Charles Sarrazin <charles@sarraz.in>
  * @author Bob van de Vijver <bobvandevijver@hotmail.com>
  * @author Kevin Schuurmans <kevin.schuurmans@freshheads.com>
+ *
+ * @method void move(Entry $entry, $newParent) Moves an entry on the Ldap server
```

Comment:
missing "string" type hint on the argument (same in the method below), isn't it?